metadata-ingestion/src/datahub/ingestion/source/ldap.py

```diff
@@ -153,31 +184,47 @@ def handle_user(self, dn: str, attrs: dict) -> Iterable[MetadataWorkUnit]:
             yield wu
         yield from []
 
+    def handle_group(
+        self, dn: str, attrs: Dict[str, Any]
+    ) -> Iterable[MetadataWorkUnit]:
+        """Creates a workunit for LDAP groups."""
+
+        mce = self.build_corp_group_mce(attrs)
+        if mce:
+            wu = MetadataWorkUnit(dn, mce)
+            self.report.report_workunit(wu)
+            yield wu
+        yield from []
+
     def build_corp_user_mce(
         self, dn: str, attrs: dict, manager_ldap: Optional[str]
     ) -> Optional[MetadataChangeEvent]:
         """
         Create the MetadataChangeEvent via DN and attributes.
         """
-        ldap = guess_person_ldap(dn, attrs)
+        ldap_user = guess_person_ldap(attrs)
         full_name = attrs["cn"][0].decode()
         first_name = attrs["givenName"][0].decode()
         last_name = attrs["sn"][0].decode()
-        email = (attrs["mail"][0]).decode() if "mail" in attrs else ldap
+        email = (attrs["mail"][0]).decode() if "mail" in attrs else ldap_user
         display_name = (
-            (attrs["displayName"][0]).decode() if "displayName" in attrs else full_name
+            (attrs["displayName"][0]).decode()
```

Comment:
I just ran black over it in the venv. It autoformatted it differently. As I wasn't sure about the line length I didn't know if I had to change it to something else.

Maybe some preset Make commands with a specific line length are also handy. We use this for all our formatting/ linting/ etc.

Reply:
That's pretty odd - we should be using the default line length with black. In general, these commands (from https://datahubproject.io/docs/metadata-ingestion/developing/#sanity-check-code-before-committing) should be all you need, and we run a similar set of commands in CI
```
black .
isort .
flake8 .
mypy .
pytest
```